libs/coin-modules/coin-filecoin/.unimportedrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ignoreUnresolved": [
+    "follow-redirects",
+    "form-data",
+    "proxy-from-env"
+  ],
+  "ignoreUnused": [
+    "@ledgerhq/devices",
```

Comment:
This means this library is not used and can be removed from coin-filecoin package.json

Reply:
`"@ledgerhq/devices";` is used in `bot-specs.ts` I am removing it from `"ignoreUnused"`